utils/metrics.py

```diff
@@ -114,82 +111,38 @@ def __init__(self, label, pred, num_classes):
 
     def update(self, metric_func):
         metric = {}
-        classes= []
+        classes = []
         for i in range(self.num_classes):
-            c_label = self.label.ravel() == i
-            if c_label.sum() == 0:
-                classes.append(np.nan)
-                continue
-            c_pred = self.pred.ravel()== i
+            c_label = self.label == i
+            c_pred = self.pred == i
+            # if c_label.sum() == 0 and c_pred.sum() == 0:
+            #     classes.append(np.nan)
+            # else:
```

Comment:
what will happen to these commented lines?

Reply:
I would delete them, I think the current behavior is preferred confirmed by Pierre